src/TokenizerDelim.cpp

```diff
@@ -160,9 +161,17 @@ Token TokenizerDelim::nextToken() {
 
     case STATE_COMMENT:
       if (*cur_ == '\r' || *cur_ == '\n') {
+
+        // If we have read at least one record on the current row go to the
+        // next row, line, otherwise just ignore the line.
+        if (col_ > 0) {
+          newRecord();
+          ++row;
```

Comment:
I'm having vague flashbacks that this is why I never fixed this earlier ;) Probaby interacts with skip too